inc/ConfigMap.h

```diff
@@ -55,29 +71,14 @@ namespace PCOE {
          **/
         void set(const std::string& key, const std::string& value);
```

Comment:
I think we should add others setters (set( key, vec<str>), set(key, int), etc...